<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="check_original">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_nk">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Noise killer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_af">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Adaptative filter</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_freq">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency</string>
   </property>
  </widget>
  <widget class="MplWidget" name="MplWidget" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>631</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_graph">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_clear">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_time">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose what to see:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_playO">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_playNK">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Noise killer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_playAF">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Adaptative filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select audio to play:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
